src/hardware/vga_s3.cpp

```diff
@@ -751,9 +751,18 @@ void filter_compatible_s3_vesa_modes()
 	}
 
 	auto mode_allowed = [&](const VideoModeBlock& m) {
-		// Allow all text modes
+		// Only allow standard text modes
 		if (m.type == M_TEXT) {
-			return true;
+			constexpr auto _132x28 = 0x230;
+			constexpr auto _132x30 = 0x231;
+			constexpr auto _132x34 = 0x232;
+
+			const std::vector disallowed_modes = {_132x28, _132x30, _132x34};
+
+			const auto is_allowed = (std::ranges::find(disallowed_modes,
+			                                           m.mode) ==
+			                         disallowed_modes.end());
+			return is_allowed;
```

Comment:
We have a `contains()` convenience template function in `support.h` that could be used?

It could become as compact as something like:

```cpp
return !contains(std::array{_132x28, _132x30, _132x34}, m.mode);
```

Reply:
Ah, cheers, forgot about that. Yeah I don't like it either how verbose this is.